src/components/DiceRoller.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div>
+        <button @click="rollDice">Roll Dice</button>
+        <h3>{{details}} <span v-html="this.results"></span></h3>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'DiceRoller',
+    props: {
+        numDice: {
+            type: Number,
+            required: true,
+            default: 1
+        },
+        dieType: {
+            type: Number,
+            required: true,
+            default: 20
+        },
+        modifier: {
+            type: Number,
+            default: 0
+        }
+    },
+    data () {
+        return {
+            results: 0
+        }
+    },
+    computed: {
+        details() {
+            let details = String(this.numDice) + 'd' + String(this.dieType)

```

Comment:
Rather than concatenating strings you can use template literals which are generally cleaner and easier to read:
```suggestion
            let details = `${this.numDice}d${this.dieType}`
```